packages/samples/atomic/index.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html dir="ltr" lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
+    <title>Atomic Components</title>
+
+    <script type="module" src="https://unpkg.com/@coveo/atomic@0.7.0-alpha.33/dist/atomic/atomic.esm.js"></script>
+    <link rel="stylesheet" href="https://unpkg.com/@coveo/atomic@0.7.0-alpha.33/dist/atomic/themes/default.css" />
```

Comment:
How will this sample be used inside CSB? 

Is hard-coding a specific version desired? Or should the atomic version reflect the latest atomic package version?

Reply:
When we soon get the CDN we can change the link for it, until then we can set latest or even alpha to unpkg.com as well.
```
<script type="module" src="https://unpkg.com/@coveo/atomic@alpha/dist/atomic/atomic.esm.js" ></script>
<link rel="stylesheet" href="https://unpkg.com/@coveo/atomic@alpha/dist/atomic/themes/default.css" />
```